LICENSE.md

```diff
@@ -1 +1,295 @@
+  - **Global**: ESRI Open Data - Daticos
+  - **Global**: Natural Earth Urban (NEU)
+  - **Global**: National Renewable Energy Laboratory (NREL)
+  - **Global**: World Resource Institute (WRI)
+  - **Global**: International Steering Committee for Global Mapping (ISCGM)
+  - **Global**: Art of the Mappable (AOTM) for Mapzen under CC0
+  - **Global**: OCHA_OPEN
+  - **Global**: USAID
+  - **Global**: Empirical Study of Conflict (ESOC)
+  - **Guinea**: OCHA HDX
+  - **Guinea**: WFP
+  - **Guinea**: International Organization for Migration
+  - **Guyana**: Guyana Bureau of Statistics (GBOS)
+  - **Honduras**: ISCGM
+  - **Honduras**: Direccion General de Catastro y Geografia
+  - **India**: DataMeet
+  - **Indonesia**: Geospatial Information Agency
+  - **Indonesia**: ISCGM
+  - **Iraq**: ESOC
+  - **Iraq**: World Food Program
+  - **Iraq**: Relief Web Humanitarian Information Center
+  - **Israel**: Revolutionary GIS
+  - **Israel**: Israel Central Bureau of Statistics (ICBS)
+  - **Jamaica**: Statistical Institute of Jamaica
+  - **Japan**: Geospatial Information Authority of Japan (GSI)
+  - **Kyrgyzstan**: OCHA
+  - **Kyrgyzstan**: Ministry of Emergency Situations of the Kyrgyz Republic
+  - **Liberia**: GIST
+  - **Liberia**: USAID
+  - **Liberia**: OFDA Economic and Humanitarian Assistance
+  - **Libya**: OCHA HDX
+  - **Libya**: Libya Bureau of Statistics and Census (BSC)
+  - **Libya**: IOM
+  - **Macedonia**: EuroGlobalMap
+  - **Mali**: OCHA
+  - **Mauritania**: OCHA
+  - **Mauritius**: Ministry of Housing and Lands
+  - **Mauritius**: ISCGM
+  - **Mozambique**: ISCGM
+  - **Mozambique**: CENACARTA
+  - **New Zealand**: Statistics New Zealand (SNZ)
+  - **Nicaragua**: ISCGM
+  - **Nicaragua**: INETER
+  - **Niger**: OCHA
+  - **Nigeria**: OCHA
+  - **Palestine**: OCHA
+  - **Palestine**: Ministry of planning (Palestinian Authority)
+  - **Papua**: ISCGM
+  - **Papua**: National Mapping Bureau
+  - **Paraguay**: DGEEC
+  - **Paraguay**: Republica de Paraguay Dirección General de Estadística, Encuestas y Censos (DGEEC)
+  - **Peru**: OCHA
+  - **Peru**: Instituto Geográfico Nacional
+  - **Rwanda**: Revolutionary GIS
+  - **Rwanda**: National Institute of Statistics of Rwanda (NISR)
+  - **Senegal**: OCHA
+  - **Serbia**: EuroGlobalMap
+  - **Serbia**: EuroGeoGraphics
+  - **South Africa**: Revolutionary GIS
+  - **South Africa**: South Africa Municipal Demarcation Board (SAMDB)
+  - **Taiwan**: National Land Surveying and Mapping Center
+  - **Tanzania**: Tanzania National Bureau of Statistics (TNBS)
+  - **Togo**: ISCGM
+  - **Togo**: Direction General of Cartography - DGC Togo
+  - **Ukraine**: Revolutionary GIS
+  - **Ukraine**: OCHA Ukraine ДНВП Картографія
+  - **Zimbabwe**: Zimbabwe Central Statistics Office (ZimStat)
+  - **Zimbabwe**: OCHA HDX
+- **Metazen**: source [data](https://www.mapzen.com/) and [license](https://github.com/whosonfirst-data/whosonfirst-data/blob/master/LICENSE.md#license)
```

Comment:
If this is true, the data repo license file needs to still include the original Mapzen note.

Reply:
What is the original Mapzen note?

Reply:
> Remember, some sources require attribution, some do not. Mapzen's original work, including the format and structure that allows Who's On First to operate, is made available under a Creative Commons Zero designation, and a shout out would be lovely.

Which I only see here now: https://whosonfirst.org/docs/licenses/

Reply:
(goes in whosonfirst-data repo LICENSE)

Reply:
This will be tracked in a whosonfirst-data issue that I'll link to this PR once closed.

Reply:
https://github.com/whosonfirst-data/whosonfirst-data/pull/1797